client/src/components/Page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import { t } from "@lingui/macro";
+import { withI18n, withI18nProps } from "@lingui/react";
+import helpers from "../util/helpers";
+
+const metadata = {
+  keywords: t`Landlord, Portfolio, Tenant, Displacement, Map, JustFix, NYC, New York, Housing, Who Owns What`,
+  description: t`Use this free tool from JustFix.nyc to research your building and investigate landlords! We use property ownership mapping to identify a landlord's portfolio and provide data to indicate potential tenant harassment and displacement.`,
+  siteName: t`Who owns what in nyc?`,
+};
+
+export const FB_APP_ID = "247990609143668";
+const TWITTER_HANDLE = "@JustFixNYC";
+const ORIGIN_URL = "https://whoownswhat.justfix.nyc/";
+const SHARE_IMAGE_URL = "https://i.imgur.com/6WL74DZ.png";
+
+type PageProps = {
+  title?: string;
+  children: React.ReactNode;
+};
+
+const Page = withI18n()((props: PageProps & withI18nProps) => {
+  const i18n = props.i18n;
+  const title = props.title && helpers.titleCase(props.title.trim());
+  const fullTitle = title
+    ? `${title} | ${i18n._(t`Who owns what in nyc?`)}`
+    : i18n._(t`Who owns what in nyc?`);
```

Comment:
This is probably outside the scope of this PR but is there a reason we're not capitalizing NYC here?  I'm wondering if it might also be more likely that a screen reader would pronounce it as letters rather than as some kind of bizarro word that sounds like "nyuk".  I dunno though.

Reply:
It's fully a styling choice (from the old days). I think we should let this sit here until the rebrand happens. And yes, you're right— VoiceOver pronounced it as "nick". That is precisely why I added the aria-label below to make it pronounceable for screen readers.